test/__snapshots__/help.test.ts.snap

```diff
@@ -0,0 +1,75 @@
+// Vitest Snapshot v1
+
+exports[`--help > snapshot 1`] = `
+"
+  [1m[36mserve[39m[22m - Static file serving and directory listing
+
+  [1mUSAGE[22m
+
+    [1m$[22m [36mserve[39m --help
+    [1m$[22m [36mserve[39m --version
+    [1m$[22m [36mserve[39m folder_name
+    [1m$[22m [36mserve[39m [-l [4mlisten_uri[24m [-l ...]] [[4mdirectory[24m]
+
+    By default, [36mserve[39m will listen on [1m0.0.0.0:3000[22m and serve the
+    current working directory on that address.
+
+    Specifying a single [1m--listen[22m argument will overwrite the default, not supplement it.
+
+  [1mOPTIONS[22m
+
+    --help                              Shows this help message
+
+    -v, --version                       Displays the current version of serve
+
+    -l, --listen [4mlisten_uri[24m             Specify a URI endpoint on which to listen (see below) -
+                                        more than one may be specified to listen in multiple places
+
+    -p                                  Specify custom port
+
+    -d, --debug                         Show debugging information
+
+    -s, --single                        Rewrite all not-found requests to \`index.html\`
+
+    -c, --config                        Specify custom path to \`serve.json\`
+
+    -C, --cors                          Enable CORS, sets \`Access-Control-Allow-Origin\` to \`*\`
+
+    -n, --no-clipboard                  Do not copy the local address to the clipboard
+
+    -u, --no-compression                Do not compress files
+
+    --no-etag                           Send \`Last-Modified\` header instead of \`ETag\`
+
+    -S, --symlinks                      Resolve symlinks instead of showing 404 errors
+    
+    --ssl-cert                          Optional path to an SSL/TLS certificate to serve with HTTPS
+    
+    --ssl-key                           Optional path to the SSL/TLS certificate's private key
+
+    --ssl-pass                          Optional path to the SSL/TLS certificate's passphrase
+
+    --no-port-switching                 Do not open a port other than the one specified when it's taken.
+
+  [1mENDPOINTS[22m
+
+    Listen endpoints (specified by the [1m--listen[22m or [1m-l[22m options above) instruct [36mserve[39m
+    to listen on one or more interfaces/ports, UNIX domain sockets, or Windows named pipes.
+
+    For TCP ports on hostname \\"localhost\\":
+
+      [1m$[22m [36mserve[39m -l [4m1234[24m
+
+    For TCP (traditional host/port) endpoints:
+
+      [1m$[22m [36mserve[39m -l tcp://[4mhostname[24m:[4m1234[24m
+
+    For UNIX domain socket endpoints:
+
+      [1m$[22m [36mserve[39m -l unix:[4m/path/to/socket.sock[24m
```

Comment:
Tests are failing because the output here differs when ran in CI - any ideas how we could work around this?

Reply:
Fixed it by forcing color output when running the tests - https://github.com/gamemaker1/serve/commit/ffcd4c90b5cf2f8c50a6af5cdc4818caca751e10

Reply:
Should we close this PR in favor your branch, as it seems to supersede this one?

Reply:
Sure!